test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -253,3 +247,104 @@ TEST_F(YacGateTest, DifferentCommit) {
 
   ASSERT_TRUE(gate_wrapper.validate());
 }
+
+/**
+ * @given yac gate with current round initialized
+ * @when vote for older round is called
+ * @then vote is ignored
+ */
+TEST_F(YacGateTest, OlderVote) {
```

Comment:
Seems that this and further tests have the same structure. I think better to move initialization logic to a new fixture based on `YacGateTest`.